horizons/util/python/decorators.py

```diff
@@ -97,12 +97,11 @@ def __call__(self, *args, **kwargs):
 					del self.cache_dates[key]
 					return self(*args, **kwargs)
 			else:
-				self.cache_dates[key] = time.time() # new entry
+				self.cache_dates[key] = time.time()  # new entry
```

Comment:
We only have a handful of lines with inline comments that are prefixed with two spaces. So this and the other change below need to go.